#! /bin/sh /usr/share/dpatch/dpatch-run
## cleanscore-prototypes.dpatch by gregor herrmann <gregoa@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Taken from https://groups.google.com/group/news.software.readers/msg/5bdbd6dfa74332b8
## DP: Original author: Ignatios Souvatzis
## DP: subroutine prototypes have changed in perl since 5.12

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' slrn-1.0.0~pre18~/contrib/cleanscore slrn-1.0.0~pre18/contrib/cleanscore
--- slrn-1.0.0~pre18~/contrib/cleanscore	2009-12-02 16:44:32.000000000 +0100
+++ slrn-1.0.0~pre18/contrib/cleanscore	2011-12-03 15:07:24.000000000 +0100
@@ -27,8 +27,8 @@
 
 sub help();
 sub reset_vars(%);
-sub insert_comment(%%);
-sub end_of_score(%$);
+sub insert_comment(%);
+sub end_of_score(%);
 sub clean_file($);
 
 # globals variables.
@@ -323,7 +323,7 @@
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-sub end_of_score(%$)
+sub end_of_score(%)
 {
   my $entry = shift;
   my $file_is_changed = shift;
@@ -355,7 +355,7 @@
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-sub insert_comment(%%)
+sub insert_comment(%)
 {
   my $comment=shift;
   my $entry=shift;
